src/lwe/iwlib.h

```diff
@@ -27,10 +27,31 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
-#include <netdb.h>		/* gethostbyname, getnetbyname */
+#include <netdb.h>		    /* gethostbyname, getnetbyname */
 #include <net/ethernet.h>	/* struct ether_addr */
 #include <sys/time.h>		/* struct timeval */
 #include <unistd.h>
+#include <stdint.h>
+
+/*
+ * This are types for compatibily
+ */
+#ifdef __linux__
+  #include "linux/types.h"
```

Comment:
maybe this can be entirely removed.
if you can compile that despite not having a header included that defines caddr_t (like mentioned in the original comment, that you removed) that's a strong hint in that direction.